libraries/AP_HAL/AP_HAL_Boards.h

```diff
@@ -18,25 +18,26 @@
 #define HAL_BOARD_SUBTYPE_NONE -1
 
 /* HAL Linux sub-types, starting at 1000 */
-#define HAL_BOARD_SUBTYPE_LINUX_NONE       1000
-#define HAL_BOARD_SUBTYPE_LINUX_ERLEBOARD  1001
-#define HAL_BOARD_SUBTYPE_LINUX_PXF        1002
-#define HAL_BOARD_SUBTYPE_LINUX_NAVIO      1003
-#define HAL_BOARD_SUBTYPE_LINUX_ZYNQ       1004
-#define HAL_BOARD_SUBTYPE_LINUX_BBBMINI    1005
-#define HAL_BOARD_SUBTYPE_LINUX_BEBOP      1006
-#define HAL_BOARD_SUBTYPE_LINUX_RASPILOT   1007
-#define HAL_BOARD_SUBTYPE_LINUX_MINLURE    1008
-#define HAL_BOARD_SUBTYPE_LINUX_ERLEBRAIN2 1009
-#define HAL_BOARD_SUBTYPE_LINUX_BH         1010
-#define HAL_BOARD_SUBTYPE_LINUX_QFLIGHT    1011
-#define HAL_BOARD_SUBTYPE_LINUX_PXFMINI    1012
-#define HAL_BOARD_SUBTYPE_LINUX_NAVIO2     1013
-#define HAL_BOARD_SUBTYPE_LINUX_DISCO      1014
-#define HAL_BOARD_SUBTYPE_LINUX_AERO       1015
-#define HAL_BOARD_SUBTYPE_LINUX_DARK       1016
-#define HAL_BOARD_SUBTYPE_LINUX_URUS       1017
-#define HAL_BOARD_SUBTYPE_LINUX_BLUE       1018
+#define HAL_BOARD_SUBTYPE_LINUX_NONE		1000
+#define HAL_BOARD_SUBTYPE_LINUX_ERLEBOARD	1001
+#define HAL_BOARD_SUBTYPE_LINUX_PXF		1002
+#define HAL_BOARD_SUBTYPE_LINUX_NAVIO		1003
+#define HAL_BOARD_SUBTYPE_LINUX_ZYNQ		1004
+#define HAL_BOARD_SUBTYPE_LINUX_BBBMINI		1005
+#define HAL_BOARD_SUBTYPE_LINUX_BEBOP		1006
+#define HAL_BOARD_SUBTYPE_LINUX_RASPILOT	1007
+#define HAL_BOARD_SUBTYPE_LINUX_MINLURE    	1008
+#define HAL_BOARD_SUBTYPE_LINUX_ERLEBRAIN2 	1009
+#define HAL_BOARD_SUBTYPE_LINUX_BH         	1010
+#define HAL_BOARD_SUBTYPE_LINUX_QFLIGHT    	1011
+#define HAL_BOARD_SUBTYPE_LINUX_PXFMINI    	1012
+#define HAL_BOARD_SUBTYPE_LINUX_NAVIO2     	1013
+#define HAL_BOARD_SUBTYPE_LINUX_DISCO      	1014
+#define HAL_BOARD_SUBTYPE_LINUX_AERO       	1015
+#define HAL_BOARD_SUBTYPE_LINUX_DARK       	1016
+#define HAL_BOARD_SUBTYPE_LINUX_URUS       	1017
+#define HAL_BOARD_SUBTYPE_LINUX_BLUE       	1018
+#define HAL_BOARD_SUBTYPE_LINUX_DRAGONBOARD     1019
```

Comment:
looks like something change the indentation, could you realignate all (with spaces, not tabs)?